<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="InteractiveView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>831</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>初始化</string>
    </property>
    <addaction name="actIni"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>重启</string>
    </property>
    <addaction name="actRestart"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>清除</string>
    </property>
    <addaction name="actClear"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actIni">
   <property name="text">
    <string>初始化残缺棋盘</string>
   </property>
   <property name="toolTip">
    <string>初始化残缺棋盘</string>
   </property>
  </action>
  <action name="actRestart">
   <property name="text">
    <string>重启程序</string>
   </property>
   <property name="toolTip">
    <string>重启程序</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>清除残缺棋盘</string>
   </property>
   <property name="toolTip">
    <string>清除残缺棋盘</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InteractiveView</class>
   <extends>QGraphicsView</extends>
   <header>interactiveview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
